concepts/string-vs-str/about.md

```diff
@@ -9,6 +9,63 @@ This document discusses how they overlap and offers some basic advice.
 `&str` is pronounced "and stir"
 `&str` is immutable.
 
+`String` is a potentially-mutable utf8-encoded representation of a sequence of Unicode codepoints. (In Java or C#, this type is called `StringBuilder`.)
+
+`&str` is a read only view of a well-formed utf8 sequence and because it's a reference it is `Copy` and can be shared.
```

Comment:
```suggestion
`&str` is a read only view of a well-formed utf8 sequence. Because it's a reference, it is `Copy` and can be shared.
```

Previous form isn't wrong, but IMO this form reads a bit better.